AudioBatchMetadata metadata
string file_path
---
VoiceActivityMarker marks
